Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'rest-client', '~> 1.6'
+gem 'rest-client', '~> 2.0'
```

Comment:
@mechanicles i think it's better to put the `>= 1.6` here. It will allow to use both 1.6, 1.8 versions (if someone needs them) and 2.0.